doc/operation/performance_tuning_tips.md

```diff
@@ -123,22 +166,47 @@ In order to calculate the appropiate capacity, just have in consideration the fo
+##<a name="section5"></a>Grouping rules
+The grouping rules feature is a powerful tool for <i>routing</i> your data, i.e. deciding the right destination (HDFS file, MySQL table, CKAN resource) for your context data; on the contrary, the default destination is used, i.e. the concatenation of the entity identifier and the entity type.
 
-Nevertheless, you may write your matching rules in a smart way:
+As you may suppose, the usage of the grouping rules is slower than using the default. This is because the destination is decided after checking a list of rules in a sequential way, trying to find a regex match. Here, worth remembering that regex matching is slow, and that you may configure as many groupin rules as you want/need.
 
-* Place the most probably matching rules first. Since the checking is sequential, the sooner the appropriate rule is found for a certain event the sooner another event may be checked. Thus, having those rules applying to the majority of the events in the first place of the list will increase the performance; then, put the rules applying to the second major set of evens, and so on.
+Nevertheless, you may write your grouping rules in a smart way:
+
+* Place the most probably grouping rules first. Since the checking is sequential, the sooner the appropriate rule is found for a certain event the sooner another event may be checked. Thus, having those rules applying to the majority of the events in the first place of the list will increase the performance; then, put the rules applying to the second major set of evens, and so on.
 * The simplest matching set of rules derive from the simplest way of naming the context entities, their types or the fiware-service they belog to (see [doc/design/interceptors.md](doc/design/interceptors.md) for more details on these concepts). Try to use names that can be easily grouped, e.g. <i>numeric rooms</i> and <i>character rooms</i> can be easily modeled by using only 2 regular expressions such as `room\.(\d*)` and `room\.(\D*)`, but more anarchical ways of naming them will lead for sure into much more different more complex rules.
 
-## Contact information
-Francisco Romero Bueno (francisco.romerobueno@telefonica.com)
-<br>
-Fermín Galán Márquez (fermin.galanmarquez@telefonica.com) 
\ No newline at end of file
+[Top](#top)
+
+##<a name="section6"></a>Writing logs
+Writing logs, as any I/O operation where disk writes are involved, is largely slow. Please avoid writing a huge number if logs unless necessary, i.e. because your are debuging Cygnus, and try running cygnus at least with `INFO` level (despite a lot of logs are still written at that level). The best is running with `ERROR` level.
+
+Logging level Cygnus run with is configured in `/usr/cugnus/conf/log4.properties`. `INFO` is configured by dafault:
+
+     flume.root.logger=INFO,LOGFILE
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/7b5b9840fcb6deefeecfc49393828e3a0ba3fb31